assets/constants.ts

```diff
@@ -4741,130 +4741,172 @@ export default {
   // stripped out to give a shorter description.
   "SUPPORTED_CONTENT_LANGUAGES": [{
```

Comment:
`SUPPORTED_CONTENT_LANGUAGES` is not used for translations we have a separate contact for translatable lanaguges. Do we need to add this over there as well?